src/Illuminate/Cache/RedisStore.php

```diff
@@ -292,10 +292,15 @@ protected function currentTags($chunkSize = 1000)
             default => '',
         };
 
+        $defaultCursorValue = match (true) {
+            $connection instanceof PhpRedisConnection => null,
```

Comment:
This probably should also do a version check as well as checking its PhpRedisConnection. That way it doesn't bread it for people who have help back on upgrading the extension
```
if (version_compare(phpversion('redis'), '6.1.0', '>=')) {
    ...
}
```

Reply:
That's a good idea. How about I do something like:

```php
$defaultCursorValue = match (true) {
    $connection instanceof PhpRedisConnection && version_compare(phpversion('redis'), '6.1.0', '>=') => null,
    default => '0',
};
```

Reply:
I'm not a maintainer, but for me this would work 😄